test/integration/default/bats/test_helper.bash

```diff
@@ -2,34 +2,31 @@
 
 echo "Setting up test disks..."
 {
-	# Unmount all disks in case they're already mounted (surpress exit code as they won't be the first time!)
-	sudo umount /media/disk1 || true 
-	sudo umount /media/disk2 || true
-	sudo umount /media/disk3 || true
+	apt-get install lsscsi  ||	sudo yum install lsscsi -y
```

Comment:
Full disclosure, this is the only bash I've ever written; it's amazing what you can piece together by copying and pasting examples from Google :)